app/src/androidTest/java/com/github/sdp/mediato/MyFollowersFragmentTest.java

```diff
@@ -104,6 +106,35 @@ public void testRecyclerViewWithTwoFollowings() {
     assertDisplayed(R.id.userAdapter_followButton);
   }
 
+  @Test
+  public void testCantFollowUnfollowItself() {
+    UserDatabase.followUser(user1.getUsername(), user1.getUsername());
+
+    sleep(1000);
```

Comment:
Replacing the time values by a final variable might be useful if we need to edit it.